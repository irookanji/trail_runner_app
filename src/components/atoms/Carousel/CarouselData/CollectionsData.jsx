import image1 from '../../../../assets/Carousel_1_1.png';
import image2 from '../../../../assets/Carousel_1_2.png';
import image3 from '../../../../assets/Carousel_1_3.png';

export const slidesCollections = [
  {
    image: image1,
    title: 'New Arrivals',
    info: 'The latest styles and limited edition colors that you can only find here (while they last, that is).',
  },
  {
    image: image2,
    title: 'Natural Run Collection',
    info: 'Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.',
  },
  {
    image: image3,
    title: 'Fresh Long Sleeve',
    info: 'It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.',
  },
  {
    image: image1,
    title: 'New Arrivals',
    info: 'The latest styles and limited edition colors that you can only find here (while they last, that is).',
  },
  {
    image: image2,
    title: 'Natural Run Collection',
    info: 'Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.',
  },
  {
    image: image3,
    title: 'Fresh Long Sleeve',
    info: 'It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.',
  },
];
