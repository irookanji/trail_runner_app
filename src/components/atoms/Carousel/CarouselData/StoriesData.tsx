import image1 from '../../../../assets/Carousel_3_1.png';
import image2 from '../../../../assets/Carousel_3_2.png';
import image3 from '../../../../assets/Carousel_3_3.png';

export const slidesStories: { image: string, title: string, info: string }[] = [
  {
    image: image1,
    title: 'Planet-Friendly Activewear',
    info: 'Run Collection is made with sustainable materials that can stand up to your workout.',
  },
  {
    image: image2,
    title: 'adidas and Allbirds',
    info: 'Learn more about FUTURECRAFT.FOOTPRINT, our collaborative shoe coming soon.',
  },
  {
    image: image3,
    title: 'How much did your outfit cost the planet?',
    info: 'Now any brand can use our Carbon Footprint tools to label their emissions.',
  },
  {
    image: image1,
    title: 'Planet-Friendly Activewear',
    info: 'Run Collection is made with sustainable materials that can stand up to your workout.',
  },
  {
    image: image2,
    title: 'adidas and Allbirds',
    info: 'Learn more about FUTURECRAFT.FOOTPRINT, our collaborative shoe coming soon.',
  },
  {
    image: image3,
    title: 'How much did your outfit cost the planet?',
    info: 'Now any brand can use our Carbon Footprint tools to label their emissions.',
  },
];
