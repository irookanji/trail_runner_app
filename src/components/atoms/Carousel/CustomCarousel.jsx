import React from 'react';
import {
  Container,
  Title,
  SubTitle,
  Description,
  SlideContainer,
  ImageContainer,
  ImageCarousel,
  ArrowButton,
} from './styles';

import arrowToRight from '../../../assets/arrow_right.svg';
import arrowToLeft from '../../../assets/arrow_left.svg';
import Carousel, { slidesToShowPlugin, arrowsPlugin } from '@brainhubeu/react-carousel';
import '@brainhubeu/react-carousel/lib/style.css';
import { v4 as uuidv4 } from 'uuid';

export default function CustomCarousel(props) {
  return (
    <Container>
      <Title>{props.titleText}</Title>
      <Carousel
        plugins={[
          'centered',
          'infinite',
          {
            resolve: arrowsPlugin,
            options: {
              arrowLeft: (
                <ArrowButton>
                  <img src={arrowToLeft} alt="Arrow To Left" name="angle-double-left" />
                </ArrowButton>
              ),
              arrowRight: (
                <ArrowButton>
                  <img src={arrowToRight} alt="Arrow To Right" name="angle-double-right" />
                </ArrowButton>
              ),
              addArrowClickHandler: true,
            },
          },
          {
            resolve: slidesToShowPlugin,
            options: {
              numberOfSlides: 3,
            },
          },
        ]}
        breakpoints={{
          640: {
            plugins: [
              'centered',
              'infinite',
              {
                resolve: slidesToShowPlugin,
                options: {
                  numberOfSlides: 1,
                },
              },
            ],
          },
          900: {
            plugins: [
              'centered',
              'infinite',
              {
                resolve: arrowsPlugin,
                options: {
                  arrowLeft: (
                    <button
                      style={{
                        background: 'none',
                        border: 'none',
                        cursor: 'pointer',
                      }}
                    >
                      <img src={arrowToLeft} alt="Arrow To Left" name="angle-double-left" />
                    </button>
                  ),
                  arrowRight: (
                    <button
                      style={{
                        background: 'none',
                        border: 'none',
                        cursor: 'pointer',
                      }}
                    >
                      <img src={arrowToRight} alt="Arrow To Right" name="angle-double-right" />
                    </button>
                  ),
                  addArrowClickHandler: true,
                },
              },
              {
                resolve: slidesToShowPlugin,
                options: {
                  numberOfSlides: 2,
                },
              },
            ],
          },
        }}
      >
        {props.slides.map((slide, index) => (
          <SlideContainer key={uuidv4()}>
            <ImageContainer>
              <ImageCarousel className="collection_img" $cover={slide.image} alt={slide.info} />
            </ImageContainer>

            <SubTitle>{slide.title}</SubTitle>

            <Description>{slide.info}</Description>
          </SlideContainer>
        ))}
      </Carousel>
    </Container>
  );
}
