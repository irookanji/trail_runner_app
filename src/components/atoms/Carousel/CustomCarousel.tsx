/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import Slider from 'react-slick';
import { v4 as uuidv4 } from 'uuid';
import {
  Container,
  Title,
  SubTitle,
  Description,
  SlideContainer,
  ImageContainer,
  ImageCarousel,
  CommonContainer,
  InfoCardContainer,
  ArrowBtn,
} from './styles';
import arrowToRight from '../../../assets/arrow_right.svg';
import arrowToLeft from '../../../assets/arrow_left.svg';
import 'slick-carousel/slick/slick.css';

type SlidesProps = {
  image: string;
  title: string;
  info: string;
}

type Props = {
  titleText: string,
  slides: Array<SlidesProps>,
};

export default function CustomCarousel({ titleText, slides }: Props) {
  const sliderSettings = {
    arrows: true,
    slidesToShow: 3,
    slidesToScroll: 1,
    infinite: true,
    prevArrow: (
      <ArrowBtn left={0}>
        <img src={arrowToLeft} alt="Arrow To Left" />
      </ArrowBtn>
    ),
    nextArrow: (
      <ArrowBtn right="100px">
        <img src={arrowToRight} alt="Arrow To Right" />
      </ArrowBtn>
    ),
    responsive: [
      {
        breakpoint: 1024,
        settings: {
          slidesToShow: 2,
        },
      },
      {
        breakpoint: 800,
        settings: {
          slidesToShow: 1,
        },
      },
    ],
  };

  return (
    <Container>
      <Title>{titleText}</Title>
      <CommonContainer>
        <Slider {...sliderSettings}>
          {slides.map((slide) => (
            <SlideContainer key={uuidv4()}>
              <ImageContainer>
                <ImageCarousel className="collection_img" $cover={slide.image} alt={slide.info} />
              </ImageContainer>
              <InfoCardContainer>
                <SubTitle>{slide.title}</SubTitle>
                <Description>{slide.info}</Description>
              </InfoCardContainer>
            </SlideContainer>
          ))}
        </Slider>
      </CommonContainer>
    </Container>
  );
}
