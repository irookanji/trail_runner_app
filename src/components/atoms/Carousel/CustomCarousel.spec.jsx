import React from 'react';
import renderer from 'react-test-renderer';
import CustomCarousel from './CustomCarousel';
import { slidesCollections } from '../../../components/atoms/Carousel/CarouselData/CollectionsData';

test('Carousel rendered as expected', () => {
  const tree = renderer.create(<CustomCarousel slides={slidesCollections} titleText="Shop The Collections" />).toJSON();
  expect(tree).toMatchSnapshot();
});
