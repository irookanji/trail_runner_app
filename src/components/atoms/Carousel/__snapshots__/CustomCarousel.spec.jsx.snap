// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel rendered as expected 1`] = `
<div
  className="sc-bdvvtL hTEDVR MuiBox-root css-0"
>
  <p
    className="MuiTypography-root MuiTypography-body1 sc-gKclnd hXhoYV css-ahj2mt-MuiTypography-root"
  >
    Shop The Collections
  </p>
  <div
    className="sc-dkPtRN cMUzjc MuiBox-root css-0"
  >
    <div
      className="slick-slider slick-initialized"
      dir="ltr"
    >
      <div
        className="sc-pVTFL bpyTJM slick-arrow slick-prev MuiBox-root css-260zpl"
        data-role="none"
        onClick={[Function]}
        style={
          Object {
            "display": "block",
          }
        }
      >
        <img
          alt="Arrow To Left"
          name="angle-double-left"
          src="arrow_left.svg"
        />
      </div>
      <div
        className="slick-list"
        onClick={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={null}
        onMouseMove={null}
        onMouseUp={[Function]}
        onTouchCancel={null}
        onTouchEnd={[Function]}
        onTouchMove={null}
        onTouchStart={[Function]}
        style={Object {}}
      >
        <div
          className="slick-track"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          style={
            Object {
              "WebkitTransform": "translate3d(0px, 0px, 0px)",
              "WebkitTransition": "",
              "msTransform": "translateX(0px)",
              "opacity": 1,
              "transform": "translate3d(0px, 0px, 0px)",
              "transition": "",
            }
          }
        >
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-3}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="The latest styles and limited edition colors that you can only find here (while they last, that is)."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg cxnBve collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    New Arrivals
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    The latest styles and limited edition colors that you can only find here (while they last, that is).
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-2}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg hgcquR collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Natural Run Collection
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-1}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg fFUzik collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Fresh Long Sleeve
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active slick-current"
            data-index={0}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="The latest styles and limited edition colors that you can only find here (while they last, that is)."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg cxnBve collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    New Arrivals
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    The latest styles and limited edition colors that you can only find here (while they last, that is).
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active"
            data-index={1}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg hgcquR collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Natural Run Collection
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active"
            data-index={2}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg fFUzik collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Fresh Long Sleeve
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={3}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="The latest styles and limited edition colors that you can only find here (while they last, that is)."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg cxnBve collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    New Arrivals
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    The latest styles and limited edition colors that you can only find here (while they last, that is).
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={4}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg hgcquR collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Natural Run Collection
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={5}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg fFUzik collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Fresh Long Sleeve
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={6}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="The latest styles and limited edition colors that you can only find here (while they last, that is)."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg cxnBve collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    New Arrivals
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    The latest styles and limited edition colors that you can only find here (while they last, that is).
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={7}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg hgcquR collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Natural Run Collection
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={8}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg fFUzik collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Fresh Long Sleeve
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={9}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="The latest styles and limited edition colors that you can only find here (while they last, that is)."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg cxnBve collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    New Arrivals
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    The latest styles and limited edition colors that you can only find here (while they last, that is).
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={10}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg hgcquR collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Natural Run Collection
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    Looking for sustainable ways to break a sweat? We’ve got you covered from head to toe.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={11}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-hKwDye fJeOmt css-1ps6pg7-MuiPaper-root"
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <div
                  className="sc-eCImPb iKawao MuiBox-root css-0"
                >
                  <div
                    alt="It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you."
                    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 sc-jRQBWg fFUzik collection_img css-1ps6pg7-MuiPaper-root"
                  />
                </div>
                <div
                  className="sc-gsDKAQ bvFUbt MuiBox-root css-0"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-iCfMLu buLcBz css-ahj2mt-MuiTypography-root"
                  >
                    Fresh Long Sleeve
                  </p>
                  <p
                    className="MuiTypography-root MuiTypography-body1 sc-furwcr jcRKPK css-ahj2mt-MuiTypography-root"
                  >
                    It’s crisp, clean and ready for wherever the day takes you. And whenever the day takes you.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-pVTFL bpyTJM slick-arrow slick-next MuiBox-root css-vcx38y"
        data-role="none"
        onClick={[Function]}
        style={
          Object {
            "display": "block",
          }
        }
      >
        <img
          alt="Arrow To Right"
          name="angle-double-right"
          src="arrow_right.svg"
        />
      </div>
    </div>
  </div>
</div>
`;
